### PR TITLE
Craft v3 compatibility

### PR DESCRIPTION
**Pull request by mikestreety**
This is an initial port of *Anchors* plugin to componers/v3 syntax.

Things I still need to do:

- [ ] Make namespace `pixelandtonic`
- [x] Update authors in `composer.json`
- [ ] Make release version `2.0.0`
- [ ] Update craft dependency
- [x] Update readme

### CODE REVIEW

File: src/twigextensions/AnchorsTwigExtension.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    /**
+     * Parses a string to automatically add anchors to all H1-H3’s
+     *
+     * @param string $html
+     * @param mixed $tags
+     * @return string
+     */
+    public function anchorsFilter($html, $tags = 'h1,h2,h3')
+    {
+        $service = new \mikestreety\anchors\services\AnchorsService;
```
Comment (mikestreety) [inline src/twigextensions/AnchorsTwigExtension.php]: Is this the right way of doing it?

File: src/translations/en/anchors.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?php
```
Comment (mikestreety) [inline src/translations/en/anchors.php]: I'm not 100% sure this file is needed

**Comment by mikestreety** [comment]
Hi guys! Sorry to bother you on this (and I know you're busy) - just wanted to put this back on your radar 😄

**Comment by takobell** [comment]
@mikestreety it's still on the radar.  We've got a new person working with us and they're ramping up. One of the first things they do will be converting our C2 plugins to C3 and will have them review this.

**Comment by mikestreety** [comment]
Great - thanks @takobell. Sorry to hassle. Will keep an eye out 😄

**Pull request closed without merging**